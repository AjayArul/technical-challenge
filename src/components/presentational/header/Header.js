import React from 'react'
import { Link } from 'react-router-dom'
import {PROJECT_NAME, MENU_HOME, MENU_COMPAIGNS} from './../../../Strings'

const Header = () => {
    return (
        <header>
            <nav className="navbar navbar-expand-lg  navbar-dark bg-info">
                <div className="container-fluid">
                    <Link className="navbar-brand" to="/">{PROJECT_NAME}</Link>
                    <button 
                        className="navbar-toggler"
                        type="button"
                        data-toggle="collapse"
                        data-target="#navBar"
                        aria-controls="navBar"
                        aria-expanded="false"
                        aria-label="nav bar"
                    >
                        <span className="navbar-toggler-icon"></span>
                    </button>
                    <div className="collapse navbar-collapse ml-4" id="navbar">
                        <ul className="navbar-nav mr-auto">
                            <li className="nav-item">
                                <Link className="nav-link" to="/">{MENU_HOME}</Link>
                            </li>
                            <li className="nav-item">
                                <Link className="nav-link" to="/compaigns">{MENU_COMPAIGNS}</Link>
                            </li>
                        </ul>
                    </div>
                </div>
            </nav>
        </header>
    )  
}

export default Header