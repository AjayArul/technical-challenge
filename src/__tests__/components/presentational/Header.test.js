import React, {Component} from 'react'
import Header from './../../../components/presentational/header/Header';

describe('Header test <Header />', () => {
  let _wrapper
  beforeEach(() => {
    _wrapper = shallow(<Header />)
  })
  it('Should Defined and render <Header /> component', () => {
      expect(_wrapper).toBeDefined()
      expect(_wrapper).toMatchSnapshot()
  });
});
