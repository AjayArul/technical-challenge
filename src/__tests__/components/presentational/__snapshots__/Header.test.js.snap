// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test <Header /> Should Defined and render <Header /> component 1`] = `ShallowWrapper {}`;
